sssd_test_framework/utils/authentication.py

```diff
@@ -364,9 +357,7 @@ def passkey_with_output(
         :param script: Path to local umockdev script file
         :type script: str
         :param command: Command that needs to check after authentication of user
-        :type command: str | Any
-        :param expected_output_from_cmd: Output we get from command
-        :type expected_output_from_cmd: str | Any
+        :type command: str = exit 0
```

Comment:
Default value should go inti `:param:`, e.g. `Command executed after user is authenticated, defaults to "exit 0"`

Reply:
updated.